app/assets/src/components/views/report/SampleDetailsSidebar/SampleDetailsSidebar.jsx

```diff
@@ -20,7 +20,7 @@ import {
   processMetadataTypes,
   processPipelineInfo,
   processAdditionalInfo
-} from "./utils";
+} from "../../../utils/metadata";
```

Comment:
Use an alias: `~/components/utils/metadata`

Reply:
I'm not sure if all of these methods should be moved into `components/utils`. Originally, they were in `SampleDetailsSidebar/utils`. `processPipelineInfo` outputs stuff in a display format specific to `SampleDetailsSidebar`. I would just move `processMetadata` into `components/utils`, as that function seems a little more generic.

Reply:
Humm... not sure I agree.
It seems like you do a lot of useful stuff there. Wherever you want to display/use this data, it will be useful.
Actually, I think this pipeline info can also eventually be useful for heatmap (compare by date, .
For instance we might want to compare samples of different pipelines.

But will refactor per need for now.